src/middlewares/error.middleware.ts

```diff
@@ -0,0 +1,33 @@
+/* eslint-disable @typescript-eslint/no-unused-vars */
```

Comment:
I think that a better approach (instead of disabling lint rules for unused vars) is to use the underscore prefix (_req, _next)